### ISSUE TITLE
Talk is not accessible for users with a visual impairment

### ISSUE THREAD

**Issue by hytsch** [nextcloud/spreed#9567]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
Talk is not accessible. Users with a visual impairment cannot use the TalkApp (browser and SmartPhoneApp). - Would you be interested in remedying this deficiency? We could support you as a tester.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by nickvergessen** [comment]
Yeah, unluckily we collected quite some issues over the past, all labeled with the accessibility label: https://github.com/nextcloud/spreed/labels/accessibility

Anything in particular that would help you as a first step or something in particular that is very bad?
